opencti-platform/opencti-graphql/src/modules/relationsRef/stixCoreObject-registrationRef.ts

```diff
@@ -1,8 +1,8 @@
 import { schemaRelationsRefDefinition } from '../../schema/schema-relationsRef';
 import { ABSTRACT_STIX_CORE_OBJECT } from '../../schema/general';
-import { createdBy, externalReferences, objectLabel, objectMarking, xOpenctiLinkedTo } from '../../schema/stixRefRelationship';
+import { createdBy, externalReferences, internalFiles, objectLabel, objectMarking, work, xOpenctiLinkedTo } from '../../schema/stixRefRelationship';
 
 schemaRelationsRefDefinition.registerRelationsRef(
   ABSTRACT_STIX_CORE_OBJECT,
-  [createdBy, objectMarking, objectLabel, externalReferences, xOpenctiLinkedTo]
+  [createdBy, objectMarking, objectLabel, externalReferences, internalFiles, work, xOpenctiLinkedTo]
```

Comment:
i don't get these new refs for stix core objects. When and how are they used? Is it in preparation of a future chunk?

Reply:
Those refs indicate the possible relations.
Adding them allowed us to add markings to those entities.

Reply:
On second thought, those refs are not needed
I remove them